submissions/ArthurGorbenko/HTML5-Form/index.html

```diff
@@ -0,0 +1,487 @@
+<html lang="en">
+  <head>
+    <link
+      href="https://fonts.googleapis.com/css?family=Open+Sans&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="node_modules/normalize.css/normalize.css" />
+    <link rel="stylesheet" href="css/styles.css" />
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Document</title>
+  </head>
+  <body>
+    <header class="page_header">
+      <button type="button" class="button page_header_button_back">
+        <h4 class="page_header_button_back_text">Go back</h4>
+      </button>
+      <img src="images/logo-header.png" alt="logo" class="header_logo" />
+      <input
+        type="button"
+        value="Sign in"
+        class="button page_header_button_forward"
+      />
+    </header>
+    <main>
+      <form action="#" class="form">
+        <section class="form-header">
+          <section class="form-header_status">
+            <div class="wrapper_form-header_status_content">
+              <h4 class="form-header_titles form-header_status_title_text">
+                Your order
+              </h4>
+              <table class="form-header_info_table">
+                <thead>
+                  <th class="form-header_status_table_cell">ITEMS</th>
+                  <th class="form-header_status_table_cell">DETAILS</th>
+                  <th class="form-header_status_table_cell">PRICE</th>
+                  <th class="form-header_status_table_cell">DELIVERY</th>
+                </thead>
+                <tbody>
+                  <tr>
+                    <td class="form-header_status_table_cell">2 documents</td>
+                    <td class="form-header_status_table_cell">2 languages</td>
+                    <td class="form-header_status_table_cell">$696.06</td>
+                    <td class="form-header_status_table_cell">48 hours*</td>
+                  </tr>
+                </tbody>
+              </table>
+            </div>
+          </section>
+            <section class="wrapper_form-header_status_buttons">
+              <button
+                type="submit"
+                class="button submit_button form-header_submit_button"
+              >
+                Place order
+              </button>
+              <button type="button" class="button form-header_print_button">
+                Print a quote
+              </button>
+            </section>
+          <section class="form-header_info">
+            <div class="wrapper_form-header_info_content">
+              <h4 class="form-header_titles form-header_info_title">
```

Comment:
The icon must be aligned vertically with respect to the title. [Image](http://prntscr.com/ocys82).

Reply:
Done.